tests/test_cli.py

```diff
@@ -930,3 +930,12 @@ def test_description(runner):
     ])
 
     assert 'Takshila' in result.output
+
+
+def test_edit_description(runner, todos, todo_factory):
+    todo_factory(summary='harhar\n', description='Parnidi')
+
+    result = runner.invoke(cli, ['edit', '1', '--description', 'Parnidi'])
```

Comment:
When editing the todo, set the description to something different than the original value. Otherwise it's unclear if the `edit` did anything or not.